sklearn/utils/metaestimators.py

```diff
@@ -70,3 +71,14 @@ def if_delegate_has_method(delegate):
     False
     """
     return lambda fn: _IffHasAttrDescriptor(fn, '%s.%s' % (delegate, fn.__name__))
+
+
+def indices_to_mask(indices, mask_length):
```

Comment:
Thinking about it, this is more relevant to validation (i.e. different input scenarios yield different data formats) than to metaestimators